typings/index.d.ts

```diff
@@ -2117,6 +2121,14 @@ declare module 'discord.js' {
     public setPermissions(
       permissions: GuildApplicationCommandPermissionData[],
     ): Promise<Collection<Snowflake, ApplicationCommandPermissions[]>>;
+    public addPermissions(
+      command: ApplicationCommandResolvable,
+      permissions: ApplicationCommandPermissionData[],
+    ): Promise<ApplicationCommandPermissions[]>;
+    public removePermissions(
+      command: ApplicationCommandResolvable,
+      userOrRole: UserResolvable | RoleResolvable | UserResolvable[] | RoleResolvable[],
+    ): Promise<ApplicationCommandPermissionData[]>;
```

Comment:
```suggestion
    ): Promise<ApplicationCommandPermissions[]>;
```

Reply:
@Sasuke-7 this should be the last requested change, and then the PR will look good to me.